diff --git a/node_modules/flareact/src/components/_document.js b/node_modules/flareact/src/components/_document.js
index 3494b60..206b493 100644
--- a/node_modules/flareact/src/components/_document.js
+++ b/node_modules/flareact/src/components/_document.js
@@ -61,6 +61,7 @@ export function FlareactHead({ helmet, page, buildManifest }) {
       {helmet.title.toComponent()}
       {helmet.meta.toComponent()}
       {helmet.link.toComponent()}
+      {helmet.script.toComponent()}
 
       {[...links].map((link) => (
         <link href={`/_flareact/static/${link}`} rel="stylesheet" />
